cpp/modmesh/view/RMainWindow.cpp

```diff
@@ -30,17 +30,136 @@
 
 #include <modmesh/view/R3DWidget.hpp>
 
+#include <modmesh/view/RAction.hpp>
+#include <modmesh/view/RApplication.hpp>
+#include <Qt>
+#include <QMenuBar>
+#include <QMenu>
+#include <QAction>
+#include <QActionGroup>
+
 namespace modmesh
 {
 
+RMainWindow::RMainWindow()
+    : QMainWindow()
+{
+    // Do not call setUp() from the constructor.  Windows may crash with
+    // "exited with code -1073740791".  The reason is not yet clarified.
```

Comment:
I troubleshooted this on the github actions CI and no way to use a debugger to debug it.

Reply:
@tychuang1211 This baffles me a lot.  Do you think you have time to see if you can reproduce this issue?